app/views/content/life-as-a-teacher/age-groups-and-specialisms/secondary/_article.html.erb

```diff
@@ -56,7 +56,7 @@
     <h4>Key stage 5 (KS5)</h4>
     <p>KS5 includes years 12 and 13, with students aged 16 to 18 studying at secondary schools with sixth forms, or at colleges.</p>
     <p>Most students in KS5 will be working towards A levels, T Levels or other equivalent qualifications. At this age, students are given the opportunity to follow their own interests. The subjects and qualifications they choose will impact what they can apply for at university, and whether they would be well placed to pursue apprenticeships or start skilled employment.</p>
-<p>If teaching secondary sounds like the career for you, find out more about <a href="/steps-to-become-a-teacher">how to become a teacher</a>.</p>
+    <%= render CallsToAction::ArrowLinkComponent.new(link_target: "/steps-to-become-a-teacher", link_text: "Find out more about how to become a teacher") %>
```

Comment:
```suggestion
    <%= render CallsToAction::ArrowLinkComponent.new(link_target: "/steps-to-become-a-teacher", link_text: "Find out how to become a teacher") %>
```

We haven't told them anything about **how** to become a teacher yet, so this doesn't need `more`?